examples/dataset/person_detection/tests/test_person_detection.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import random
+import string
+from argparse import Namespace
+from collections.abc import Iterator
+
+import pytest
+from person_detection.upload_dataset import run as upload_dataset_main
+from person_detection.upload_results import run as upload_results_main
+
+from kolena._utils.state import kolena_session
+
+
+DATASET = "coco-person-detection"
```

Comment:
`DATASET = "coco-2014-val-person-detection"` duplicate or intentional?

Reply:
Updated (and actually running this test now)